src/mpyl/plan/discovery.py

```diff
@@ -20,17 +20,18 @@ def find_projects() -> list[Path]:
     # is a bit harder to read but completes in a couple of seconds.
     # If you find a faster way to list all files, please raise a PR.
 
-    command = (
-        f"find {Path('.')}"
-        " -type d ( -name target -o -name .git ) -prune"
-        " -o ("
-        f" -path **/deployment/{Project.project_yaml_file_name()}"
-        f" -o -path **/deployment/{Project.project_overrides_yaml_file_pattern()}"
-        " )"
-        " -print"
-    )
+    command = [
+        "fdfind",
+        "--glob",
+        "-E",
```

Comment:
Nitpick: it's usually a good idea to use the long version for scripts, it makes it easier for the next developer. It also keeps consistency with the other long ones 🙂 

```suggestion
        "--exclude",
```